pulp_container/app/viewsets.py

```diff
@@ -586,7 +586,7 @@ class ContainerPushRepositoryViewSet(TagOperationsMixin, ReadOnlyRepositoryViewS
                 "principal": "authenticated",
                 "effect": "allow",
                 "condition": [
-                    "has_model_or_obj_perms:container.modify_content_containerrepository",
+                    "has_model_or_obj_perms:container.manage_tags_containerpushrepository",
```

Comment:
should we have this as granular as possible or rather generalize in this case? for example we could a generic permission
`modify_content_containerrepository` that would allow any content modification whether tagging, or some other action that involves content alteration  for push *and* normal repos.